### ISSUE TITLE
Feature request: Auto save and remove new tabs for images

### ISSUE THREAD

**Issue by tanmaypandey7** [abreheret/PixelAnnotationTool#32]
Auto save will be very helpful for fast annotations. Also, new tabs for images start to get very annoying if you have a large number of images. You have to close them a lot of times and if the image name is very large, they start to fill up just after 6-7 images.

**Comment by abreheret** [comment]
There are already practical shortcuts : 
- CTRL + S to save the current image
- CTRL + W close the current image

**Comment by tanmaypandey7** [comment]
Yes, I know but when I have about 300 images to annotate it becomes cubersome to keep pressing ctrl+s every time

**Comment by tanmaypandey7** [comment]
![screenshot 32](https://user-images.githubusercontent.com/36691630/53249224-9d862f80-36dd-11e9-9f2c-11731a9be504.png)
Screenshot for an image I am annotating right now. Brush is a little small for this.

**Comment by abreheret** [comment]
It's better than nothing :D  (for the shortcuts)

You do not have to open all the images at the same time, and as soon as you have finished an image you can save it and close it, then make the following image.

I leave open this issue, if one day I have time to add this feature ...

**Comment by tanmaypandey7** [comment]
Okay, can anything be done for the brush size?
Thank you for fast reply

**Comment by abreheret** [comment]
What is the problem with the 'brush size' ? 
You can change the size with the 'Circle size' under the 'Resize factor'

**Comment by tanmaypandey7** [comment]
Yes, the maximum size "99" is a bit small for high dimensional images

**Comment by abreheret** [comment]
Which max size do you think is correct ? 300 ?

**Comment by tanmaypandey7** [comment]
Apologies sir, I was using the watershed incorrectly. Instead of labelling just the borders of the target, I was coloring the entire region of interest. It is working as it is, sorry for the trouble.
